docs/conversion/sakai_21_0-21_1_mysql_conversion.sql

```diff
@@ -9,3 +9,36 @@ ALTER TABLE rbc_rating MODIFY title varchar(255) NOT NULL;
 UPDATE rbc_rating SET points = 0 WHERE points IS NULL;
 ALTER TABLE rbc_rating MODIFY points double NOT NULL;
 -- END SAK-45137
+
+-- START SAK-45601
+CREATE TABLE TINCANAPI_EVENT (
+EVENT VARCHAR(255) NOT NULL,
```

Comment:
Does this need to be 255 chars?

Reply:
Should use as small as possible to make life easier for indexes (e.g., utf8mb4 is 4 bytes per char = 255*4

Reply:
No problem, do you want a specific value? I think 64 would be enough. Does it seem right to you?

Reply:
Is it the same as SAKAI_EVENT->EVENT? That is varchar(32). Or is this different? Then 64 seems fine

Reply:
Done